util/params/params.go

```diff
@@ -0,0 +1,21 @@
+package params
```

Comment:
This is a Workflow specific util, so it shouldn't be at the top-level util folder. Can you add this to the `workflow/common` package? You can add a new file there, but it should still be under the `common` package.